src/python/TaskWorker/MasterWorker.py

```diff
@@ -313,10 +313,10 @@ def _externalScheduling(self, limit):
             # Log the formatted table
             self.logger.info('\n%s', table)
 
-            if self.config.TaskScheduling.dry_run:
-                return selected_tasks #dry_run True (with Task Scheduling)
+            if self.config.TaskWorker.task_scheduling_dry_run:
```

Comment:
beware. THis will raise an exception it the parameter is missing in the config. file.
Since this is really an optional thing, it may be better to be prepared for it to be missing and keep the config. file cleaner.